delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/client/ServiceDescription.scala

```diff
@@ -0,0 +1,24 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.client
+
+import io.circe.Decoder
+
+/**
+  * Information about the deployed service
+  *
+  * @param version service version
+  */
+final case class ServiceDescription(version: String) {
```

Comment:
The service description could be in the sdk no ? As we also have one for Blazegraph ?